os/windows/sign_azure.ps1

```diff
@@ -0,0 +1,33 @@
+# Signing script for Azure Code Signing
+# Arguments: sign_azure.ps1 path_to_sign
+#
+# Environment variables must be set up before use:
+#
+# AZURE_TENANT_ID
+# AZURE_CLIENT_ID
+# AZURE_CLIENT_SECRET
+# AZURE_CODESIGN_ACCOUNT_NAME
+# AZURE_CODESIGN_ENDPOINT
+# AZURE_CODESIGN_PROFILE_NAME
```

Comment:
In other scripts we check if something like `AZURE_CLIENT_ID` is set, and if not, print a warning about this being a fork, and silently quit. That way the `continue-on-error` can also be removed, meaning the workflow actually fails if signing would fail. Has worked out for other repos/places at least :)

Reply:
Will check what is used elsewhere and do something similar here.